### DISCUSSION TITLE
Logic Apps Standard and dotnet 8

### DISCUSSION THREAD

**Discussion by timjansenortec** [Azure/logicapps#1032] [General]
Is it already possible to use dotnet 8 for Logic App Standard?

I am using a nuget based logicapp with custom connectors written in c#.
https://learn.microsoft.com/en-us/azure/logic-apps/create-single-tenant-workflows-visual-studio-code#convert-your-project-to-nuget-package-based-net

Dotnet 8 introduces the new function isolated worker model. What is the impact of this on existing apps?
https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=windows
Or should we wait until in-process is available for dotnet 8?
https://techcommunity.microsoft.com/t5/apps-on-azure-blog/net-on-azure-functions-march-2024-roadmap-update/ba-p/4097744
https://github.com/Azure/azure-functions-host/issues/9951

Any info or guidance would be appreciated.

**Comment by mikoskinen** [comment]
Any updates on this?

**Comment by timjansenortec** [comment]
I have successfully updated our Logic App Standard with custom connectors to dotnet 8.

I changed this setting (described in documentation):
    FUNCTIONS_WORKER_RUNTIME: 'dotnet'

And added this setting:
    FUNCTIONS_INPROC_NET8_ENABLED: '1'
to keep running in-process like was done for dotnet 6. I have not tried running out-of-process.

and I used this workaround in the csproj:
```
  <!-- workaround for: https://github.com/Azure/azure-functions-host/issues/10575 -->
  <ItemGroup>
    <FunctionsPreservedDependencies Include="System.Memory.Data.dll" />
  </ItemGroup>
```
